modules/postgres/testdata/docker-entrypoint-ssl.bash

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+set -Eeo pipefail
+
+# Hardcoded user id from pg11-alpine image
```

Comment:
I'll see if I can extract the `uid` from `/etc/passwd` instead of hardcoding it here. 
At somepoint we have to make an assumption about the image . i think its safer to assume postgres runs under the `postgres` user rather than always `uid` 70